vdom/core.py

```diff
@@ -72,28 +72,37 @@ class VDOM(object):
     >>> from vdom.helpers import h1
     >>> h1('Hey')
     """
-    # This class should only have these 4 attributes
-    __slots__ = ['tag_name', 'attributes', 'children', 'key', '_frozen']
+    # This class should only have these 5 attributes
+    __slots__ = ['tag_name', 'attributes', 'style', 'children', 'key', '_frozen']
 
-    def __init__(self, tag_name, attributes=None, children=None, key=None, schema=None):
+    def __init__(self, tag_name, attributes=None, style=None, children=None, key=None, schema=None):
```

Comment:
When adding a constructor argument to an already established class I tend to add it to the end of the list to avoid positional constructor issues for pre-existing callers.

Reply:
I agree generally, but in this case:

1. We have made no releases since introducing the current signature
2. The current signature is a full break from the previous one

So I think it's ok to put this where it is, since I think those two are more related than children. 

If you feel strongly about it I'm happy to change it :)

Reply:
I meant you're removing elements from an argument dict that a caller may not expect to be mutated. Usually you want to avoid this side-effect for a function and either copy the dict first or make a new dict without the key when removing.

Reply:
No strong feelings, just thought I'd point it out if you hadn't thought about it